chore(): task#1805

SNKz0610 opened the pull request:
Adding Android Homeworks 18/05/2021 about Lmao Minh game.
Closes task#1805

SNKz0610 committed:
chore(): task#1805
Adding Android Homeworks 18/05/2021 about Lmao Minh game.
Closes task#1805

linhvnguyen9 reviewed (approved):
Bài của b khá ok đó! 1 góp ý nhỏ là b merge pull request này vào develop, sau đó mỗi bài mới chỉ cần tạo thêm các nhánh từ develop thôi, ko cần phải khởi tạo 1 project mới nữa. Như vậy thì b đỡ mất thời gian tạo project, và t review cũng sẽ nhanh hơn

JKhanh commented:
Trong đề bài có ghi là mỗi tướng có 1 bộ chiêu thức QWER đặc trưng. Nhưng theo code của bạn thì mọi đối tượng Champion đều có thực hiện giống nhau

SNKz0610 committed:
chore(): task#1805
Fixing function in abstrac class Champion to make all the champion's skills be different  at each other.
Closes task#1805
